textbook/12/1/uniform.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>If think if you underline or bold the terms "empirical distribution" and "probability distribution" it will help to remind readers the difference and relationship between the two, which they could look into to better construct an understanding and refer to more information in the probability chapter/lectures</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>